tests/fixture/e2e/flags.go

```diff
@@ -66,6 +71,12 @@ func RegisterFlags() *FlagVars {
 		false,
 		"[optional] whether to target the existing network identified by --network-dir.",
 	)
+	flag.UintVar(
+		&vars.networkShutdownDelay,
+		"network-shutdown-delay",
+		12, // Make sure this value takes into account the scrape_interval defined in scripts/run_prometheus.sh
+		"[optional] the number of seconds to wait before shutting down the test network at the end of the test run. A value greater than the scrape interval is suggested. 0 avoids waiting for shutdown.",
+	)
```

Comment:
Will need to update some types in a few other places - but I think this would be a good change
```suggestion
	flag.DurationVar(
		&vars.networkShutdownDelay,
		"network-shutdown-delay",
		12*time.Second, // Make sure this value takes into account the scrape_interval defined in scripts/run_prometheus.sh
		"[optional] the duration to wait before shutting down the test network at the end of the test run. A value greater than the scrape interval is suggested. 0 avoids waiting for shutdown.",
	)
```